changelogs/current.yaml

```diff
@@ -26,6 +26,9 @@ removed_config_or_runtime:
 - area: overload manager
   change: |
     removed ``envoy.reloadable_features.overload_manager_error_unknown_action`` and legacy code paths.
+- area: resource_monitors
+  change: |
+    removed ``envoy.reloadable_features.do_not_count_mapped_pages_as_free`` and legacy code paths.
```

Comment:
Shouldn't this be `envoy.reloadable_features.count_unused_mapped_pages_as_free`?

Reply:
Ah, the unfortunate issue when you copy and paste things from old PRs. Fixed.